pom.xml

```diff
@@ -67,11 +67,6 @@
     <tag>HEAD</tag>
   </scm>
 
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
More correctly, credentials 2.1.x requires 1.609 as the baseline, hence the need to jump